### ISSUE TITLE
Doesn't run

### ISSUE THREAD

**Issue by hellishvictor** [zer0-x/cwitch#9]
Hi, I had install it with: `pip install https://github.com/zefr0x/cwitch/archive/refs/tags/v0.3.0.zip`, and it went well.
Now, if I enter just "**cwitch -h**" it returns:
```
Traceback (most recent call last):
  File "D:\Aplicaciones (x64)\Python\Scripts\cwitch-script.py", line 33, in <module>
    sys.exit(load_entry_point('cwitch==0.3.0', 'console_scripts', 'cwitch')())
  File "D:\Aplicaciones (x64)\Python\Scripts\cwitch-script.py", line 25, in importlib_load_entry_poi
nt
    return next(matches).load()
  File "d:\aplicaciones (x64)\python\lib\importlib\metadata.py", line 77, in load
    module = import_module(match.group('module'))
  File "d:\aplicaciones (x64)\python\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "d:\aplicaciones (x64)\python\lib\site-packages\cwitch\cli.py", line 9, in <module>
    from . import prompts
  File "d:\aplicaciones (x64)\python\lib\site-packages\cwitch\prompts.py", line 31, in <module>
    ) -> tuple[tuple[int, ...], bool, Optional[int]]:
TypeError: 'type' object is not subscriptable

```
I'm on Win7 SP1 x64 using Python v3.8.2.
Cheers.

**Comment by zefr0x** [comment]
This `cwitch` project uses a new python feature, your python version is not supported.

> Deprecated since version 3.9: [builtins.tuple](https://docs.python.org/3/library/stdtypes.html#tuple) now supports subscripting ([]). See [PEP 585](https://peps.python.org/pep-0585/) and [Generic Alias Type](https://docs.python.org/3/library/stdtypes.html#types-genericalias).

See: https://docs.python.org/3/library/typing.html#typing.Tuple

---

By the way, it is highly recommended to **avoid using Windows 7** due to its lack of support and numerous security vulnerabilities. You should consider using a newer Windows version, or even better, moving to Linux if your device is not capably of doing so.

**Comment by hellishvictor** [comment]
> This `cwitch` project uses a new python feature, your python version is not supported.
> 
> > Deprecated since version 3.9: [builtins.tuple](https://docs.python.org/3/library/stdtypes.html#tuple) now supports subscripting ([]). See [PEP 585](https://peps.python.org/pep-0585/) and [Generic Alias Type](https://docs.python.org/3/library/stdtypes.html#types-genericalias).
> 
> See: https://docs.python.org/3/library/typing.html#typing.Tuple

Ok, thank you.

> By the way, it is highly recommended to **avoid using Windows 7** due to its lack of support and numerous security vulnerabilities. You should consider using a newer Windows version, or even better, moving to Linux if your device is not capably of doing so.

I won't feel more secure using that spyware called Win10/11 beyond a virtual machine.
I just want a cli tool that list the vods availables from a specified channels on twitch, like "program.exe _channelname_", so I'll keep looking.

**Comment by hellishvictor** [comment]
Solved with [Python 3.9+ installers that support Windows 7 SP1 and Windows Server 2008 R2 ](https://github.com/adang1345/PythonWin7).
Now it runs without problems, is what I needed :)
